server/controllers/upload.js

```diff
@@ -0,0 +1,71 @@
+const Submission = require("../models/Submission");
+const asyncHandler = require("express-async-handler");
+const aws = require('aws-sdk');
+const multerS3 = require( 'multer-s3' );
+const multer = require('multer');
+const path = require( 'path' );
+
+const s3 = new aws.S3({
+	accessKeyId: process.env.S3_ACCESSKEYID,
+	secretAccessKey: process.env.S3_ACCESSKEY,
+	Bucket: process.env.S3_BUCKET
+});
+
+// only allowing jpeg,jpg,png,gif file types
+const checkFileType = ( file, cb ) => {
+	const filetypes = /jpeg|jpg|png|gif/;
+	const extname = filetypes.test( path.extname( file.originalname ).toLowerCase());
+	const mimetype = filetypes.test( file.mimetype );
+	if( mimetype && extname ){
+		return cb( null, true );
+	} else {
+		cb( 'Error: Images Only!' );
+	}
+}
+
+const uploadsContestGallery = multer({
```

Comment:
👍🏽

Reply:
@rajivtitus I removed submission object creation logic. Therefore, post request to /uploads/images would not create submission object and will only return urls of the images from s3.

Reply:
Sounds great! You can merge this branch and please convey the same information to your team. Thanks!

Reply:
Thank You. I already shared the s3 credentials in the slack group and also the info.